### PR TITLE
Feed Frontend: style fixes and render nested comments

### PR DESCRIPTION
**Pull request by anniegiang**
- Fixed sizing and positioning of elements across the Feed
- The Create Post icon floats while scrolling the feed
- Fixed font weight issues by adding more weights when importing Poppins
- Incorporated copy-to-clipboard functionality using a react library (onClick it will show a modal alerting "Link Copied!")
-  Installed `antd` in order to use the Comment component
- Added like/unlike functionality
- The count for share will increase only once onClick
- Clicking comments will show comments
- Rendered nested comments (still need to style them)

WIP:
- Styling comments
- Add functionality to reply to comment
- Once the backend for auth, posts, comments, and likes are done I'll work on API integration

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "typeface-roboto";
```
Comment (robinv85) [inline client/src/index.js:3]: out of scope here but seems like we still have to remove this one

File: client/src/styles/theme.scss [hunk 2]
```diff
@@ -1,8 +1,8 @@
 // Import Colors
 @import "./colors.scss";
-@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap");
```
Comment (robinv85) [inline client/src/styles/theme.scss:3]: didn't we also have an npm dependency for the Poppins font?

Comment (anniegiang) [reply client/src/styles/theme.scss:3]: Yea it's called `typeface-poppins`. Before adding the weight values the bold text were off so not sure if the dependency works ...

Comment (luxumbra) [reply client/src/styles/theme.scss:3]: I'm glad to see @anniegiang using the @import for the google fonts as I've just added Work Sans and DM Sans to the index.scss file - I'll move it to theme.scss. I'll remove poppins dependency in my work for #88

**Pull request merged**